content/webapp/components/WorkDetails/WorkDetails.DownloadItem.tsx

```diff
@@ -0,0 +1,143 @@
+import { FunctionComponent } from 'react';
+import {
+  TransformedCanvas,
+  CustomContentResource,
+} from '@weco/content/types/manifest';
+import {
+  Body,
+  ContentResource,
+  ChoiceBody,
+  InternationalString,
+} from '@iiif/presentation-3';
+import {
+  getLabelString,
+  getFileSize,
+  isChoiceBody,
+} from '@weco/content/utils/iiif/v3';
+import styled from 'styled-components';
+import { font } from '@weco/common/utils/classnames';
+import Icon from '@weco/common/views/components/Icon/Icon';
+import { file, imageFile } from '@weco/common/icons';
+import { controlDimensions } from '@weco/content/components/ArchiveTree/ArchiveTree.helpers';
+
+export const DownloadTable = styled.table.attrs({ className: font('intr', 6) })`
+  position: relative;
+  height: ${`${controlDimensions.controlHeight}px`};
```

Comment:
same here with the double ${} wrapping, I'm just curious if it's a Prettier/linting thing?

Reply:
think it's more of an unquestioning copy/paste thing - I've changed them